co-simulation/lib/mosaic-carma-utils/src/main/java/gov/dot/fhwa/saxton/CarmaV2xMessageReceiver.java

```diff
@@ -105,18 +108,20 @@ public void run() {
            } catch (IllegalArgumentException parseError) {
                log.warn("CarmaV2xMessageReceiver received malformed message with length: " + msg.getLength() + " from client " + msg.getAddress().toString() + "! Reason: " + parseError.getMessage() +". Discarding...");
            }
-       }
+        }
+        if (listenSocket != null) {
```

Comment:
This is redundant call. If it is not running, it must be called to stop(). The socket is release in the stop() function already.

Reply:
Will address